app/index.html

```diff
@@ -24,7 +25,18 @@
 </head>
 <body>
   <div class="content with-diagram" id="js-drop-zone">
-    <div id="canvas"></div>
+    <div id="canvas">
+      <div class="git-info" >
+          <a class="icon-github-circled" href="https://github.com/bptlab/chor-js-demo" target="_blank"
+             title="Contribute on GitHub"></a>
+      </div>
+    </div>
+    <div id="panel-toggle">
```

Comment:
In Firefox, the vertical align of the > symbol is not really centered.

Reply:
For me it looks centered can you attache a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/17142708/74325945-d05f8280-4d89-11ea-874d-da2eeb0ef007.png)
Might be just my version, would not be that bad if it stayed like that.